client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { ThankYou } from 'calypso/components/thank-you';
+import theme from 'calypso/my-sites/marketplace/theme';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getTitanEmailUrl } from 'calypso/lib/titan';
+import Gridicon from 'calypso/components/gridicon';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
+type TitanSetupThankYouProps = {
+	domainName: string;
+	emailAddress?: string;
+};
+
+export const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
+	const currentRoute = useSelector( getCurrentRoute );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const translate = useTranslate();
+	const thankYouImageSrc = '/calypso/images/upgrades/thank-you.svg';
+
+	const { domainName, emailAddress = `youremail@${ domainName }` } = props;
```

Comment:
I'm not a big fan of showing `youremail@` when the email address is not available, as it's not localized. We could simply show `You email is now ready to use` in the header of the page when that's the case.